sentry/src/main/java/io/sentry/cache/IEnvelopeCache.java

```diff
@@ -6,10 +6,17 @@
 
 public interface IEnvelopeCache extends Iterable<SentryEnvelope> {
 
+  @Deprecated
   void store(@NotNull SentryEnvelope envelope, @NotNull Hint hint);
 
+  default boolean storeEnvelope(@NotNull SentryEnvelope envelope, @NotNull Hint hint) {
```

Comment:
could you also explicitly implement it in `AndroidEnvelopeCache`? I recall there were some issues with desugaring on Unity, where it would crash with NoSuchMethodError for default methods in core `sentry`